vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/server.go

```diff
@@ -117,17 +117,15 @@ cluster's shared state through which all other components interact.`,
 			verflag.PrintAndExitIfRequested()
 
 			if len(s.OpenShiftConfig) > 0 {
+				// if we are running openshift, we modify the admission chain defaults accordingly
+				admissionenablement.InstallOpenShiftAdmissionPlugins(s)
```

Comment:
no action needed, just curious: the patching function assignment was noop here, just a side effect elsewhere, so is there anything to this admission plugins install move?

Reply:
> no action needed, just curious: the patching function assignment was noop here, just a side effect elsewhere, so is there anything to this admission plugins install move?

doing it early made it easier for me to read and reason about. nothing special.